requirements.txt

```diff
@@ -7,6 +7,9 @@ colorama
 cray-product-catalog
 PyYAML
 requests-retry-session
+python-keycloak
+requests-oauthlib
```

Comment:
Since this is so closely tied to the requests package, we should instead specify that we'll use the system copy, later in the file.
```suggestion
```

Below the line `#system_python:requests`, just add the line `#system_python:requests-oauthlib`